### ISSUE TITLE
Incorrect parsing of if expressions

### ISSUE THREAD

**Issue by exerro** [exerro/Flux#2]
The following code,
```
if (a)
        if (b) {
                do c;
        }
```
Will be parsed as:
```
if (a if b) {
        do c;
}
```
Rather than the (more expected)
```
if (a) {
        if (b) {
                do c;
        }
}
```

A possible solution is to require `if` for ternary expressions to be on the same line as the true case.

**Comment by demhydraz** [comment]
Does fuckety poop Flux still have this issue?

**Comment by exerro** [comment]
Yeah, I'm more focused on the IR right now. I'll get back to it later on, along with a few other syntax tweaks and additions.

As a side note, when I said "fuckety poop Flux has an issue...", it was meant as a "fuckety poop, Flux has an issue..."
